src/modules/users/application/controllers/__tests__/user.controller.reset-password.spec.ts

```
@@ -0,0 +1,353 @@
+        newPassword: 'NewSecurePassword123!',
+      };
+      const adminPayload: JwtPayload = {
+        userId: 'admin-123',
+        email: 'admin@example.com',
+      };
+
+      const complexRequest = {
+        ip: '10.0.0.1',
+        socket: { remoteAddress: '10.0.0.2' },
+        get: jest.fn((header: string) => {
+          if (header === 'User-Agent') 
```

ESLint

[error] Delete `·`

(prettier/prettier)